docs/DIVIDING_DELPHI_CODEBASES.md

```diff
@@ -0,0 +1,60 @@
+# Dividing Delphi codebases into Sonar projects
+
+Delphi codebases can often have indistinct boundaries, with the same source files
```

Comment:
I think "indistinct" is a better word than "fuzzy" here.

Reply:
Sure, that's fair - and I'm not opinionated enough between the 2 options to mind.

I do still like "common" over "the same" because I think the term has a stronger word association for developers - especially developers working in the types of large monoliths or legacy codebases where this is likely to be the case.

Reply:
Yep, definitely agree that "common"'s the right call.